general.el

```diff
@@ -2636,6 +2636,19 @@ Return something like '((some-command-to-autoload . command) ...)."
                         :package ',name)))
                  args))))
 
+  ;; ** :general-config Keyword
+  (setq use-package-keywords
+        (cl-loop for item in use-package-keywords
+                 if (eq item :config)
+                 collect :config and collect :general-config
+                 else
+                 ;; don't add duplicates
+                 unless (eq item :general-config)
+                 collect item))
+  (defalias 'use-package-normalize/:general-config
+    #'use-package-normalize/:general)
+  (defalias 'use-package-handler/:general-config #'use-package-handler/:general)
```

Comment:
Once this goes live I will make sure to remove my similar lines, but say there is a window of time between this going live and my changing it, during that time, will there be a major issue that I kept my lines as well? It seems like this prevents adding duplicates, so it won't add any if it sees mine, but just figured I'd ask anyway since I'm not too familiar with `cl-loop`.

Reply:
You can pull the repo with straight and then remove your config before loading your config, but even if you don't, there should be no issues. It will not add a duplicate.